Renewal/System/OngoingQuestInfoList_True_EN.lub

```diff
@@ -25123,20 +25226,20 @@ QuestInfoList = {
 		CoolTimeQuest = 1
 	},
 	[8541] = {
-		Title = "No content",
+		Title = "Nothing to See",
 		IconName = "ico_dq.bmp",
 		Description = {
-			"<NAVI>Nilem<INFO>glast_01,191,273,0,101,0</INFO></NAVI> asks you to hunt 20 contaminated raidricks and 20 contaminated raidric archers and check their contents."
+			"<NAVI>[Mage Nillem]<INFO>glast_01,190,273,0,101,0</INFO></NAVI> asked you to hunt 20 Hunt Contaminated Raydrics and 20 Contaminated Raydric Archers to gather more information."
 		},
-		Summary = "Contaminated Raidrick, Decontaminated Raidrick Archer"
+		Summary = "Hunt 20 Contaminated Raydrics and 20 Contaminated Raydric Archers."
 	},
 	[8542] = {
-		Title = "Standby",
+		Title = "Nothing to See - Wait",
 		IconName = "ico_dq.bmp",
-		Description = {
-			"This quest is a daily quest, and can be performed by receiving <NAVI>Nilem<INFO>glast_01,191,273,0,101,0</INFO></NAVI> when the waiting time expires."
+		Description = { 
+			"This is a daily quest, you can take it again tomorrow when the timer expires. Speak with <NAVI>[Mage Nillem]<INFO>glast_01,190,273,0,101,0</INFO></NAVI>."
 		},
-		Summary = "Resets at 04:00",
+		Summary = "Reset at 04:00.",
```

Comment:
Why modify "Resets at 04:00"?
This is a phrase used thousands of times throughout the file.